.doc_gen/templates/zonbook/service_chapter_template.xml

```diff
@@ -34,13 +58,33 @@
     {{- with $cat_examples := index $.CategorizedExampleSets $category}}
     <section id="{{$doc_id}}_{{$cat_examples.CategoryIdSuffix}}" role="topic">
         <info>
-            <title id="{{$doc_id}}_{{$cat_examples.CategoryIdSuffix}}.title">{{$cat_examples.CategoryName}} examples for {{$.ServiceEntity.Short}}</title>
-            <titleabbrev id="{{$doc_id}}_{{$cat_examples.CategoryIdSuffix}}.titleabbrev">{{$cat_examples.CategoryName}} examples</titleabbrev>
+            <title id="{{$doc_id}}_{{$cat_examples.CategoryIdSuffix}}.title">{{$cat_examples.CategoryName}} for {{$.ServiceEntity.Short}}</title>
+            <titleabbrev id="{{$doc_id}}_{{$cat_examples.CategoryIdSuffix}}.titleabbrev">{{$cat_examples.CategoryName}}</titleabbrev>
             <abstract>
                 <para>The following code examples show how to use {{$.ServiceEntity.Short}} with &AWS; SDKs.</para>
             </abstract>
         </info>
+        {{- if eq $cat_examples.CategoryName "Cross-service examples"}}
+        <para>The following sample applications use &AWS; SDKs to combine {{$.ServiceEntity.Short}}
+            with other &AWS; services. Each example includes a link to GitHub where you can find
+            instructions on how to set up and run the application.</para>
+        {{- else if eq $cat_examples.CategoryName "Scenarios"}}
+        <para>The following code examples show how to implement common scenarios in {{$.ServiceEntity.Short}}
+            with &AWS; SDKs. These scenarios illustrate ways you can write code to use
+            an &AWS; SDK to accomplish a specific task. Each scenario includes a link to GitHub where you can find
+            instructions on how to set up and run the code.</para>
+        {{- else if eq $cat_examples.CategoryName "Actions"}}
+        <para>The following code examples show how to perform individual {{$.ServiceEntity.Short}}
+            actions with &AWS; SDKs. These excerpts illustrate how to write code
+            to call the {{$.ServiceEntity.Short}} API and are not intended to be run in
+            isolation. Each example includes a link to GitHub where you can find
+            instructions on how to set up and run the code in context.</para>
```

Comment:
<para>The following code examples demonstrate how to perform individual {{$.ServiceEntity.Short}}
            actions with &AWS; SDKs. These excerpts call the {{$.ServiceEntity.Short}} API and are not intended to be run in
            isolation. Each example includes a link to GitHub, where you can find
            instructions on how to set up and run the code in context.</para>

Reply:
Done